modules/role/manifests/st2.pp

```diff
@@ -22,4 +20,12 @@
       include ::profile::hubot
     }
   }
+
+  if $_enterprise_token {
+    include ::profile::st2flow
+    include ::profile::st2rbac
+    include ::profile::enterprise_auth_backend_ldap
+  } else {
+    include ::profile::auth_backend_pam
```

Comment:
Nope (for the better or worse) you can only use a single auth backend right now.

As far as making LDAP a pre-requisite for enterprise version, I'm a big -1 on that (LDAP should be an option available to enterprise customers, but not a requirement). It's likely that not every enterprise user uses LDAP and requiring LDAP, if nothing else, increases barrier to entry (testing).